index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
Could you please use the base template at `templates/base.html` and extend that with your body content instead of creating the whole structure?

Reply:
done.